scripts/context_retrieval/wrangle_llama_index_data.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def upload_to_gcs(bucket_name, source_file_path, destination_blob_name):
</pre><p>Shouldn't you put this type of re-usable code in a re-usable set of functions under scripts? You are going to have to copy pasta this a lot.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>